frontend/packages/dev-console/src/utils/__tests__/shared-submit-utils.spec.ts

```diff
@@ -0,0 +1,42 @@
+import * as _ from 'lodash';
+import { mockFormData } from '../../components/import/__mocks__/import-validation-mock';
+import { mockDeployImageFormData } from '../../components/import/__mocks__/deployImage-validation-mock';
+import { createService, createRoute } from '../shared-submit-utils';
+import { GitImportFormData, DeployImageFormData } from '../../components/import/import-types';
+
+describe('Shared submit utils', () => {
+  describe('Create Service', () => {
+    it('should set the correct ports in service object', () => {
+      const mockData: GitImportFormData = _.cloneDeep(mockFormData);
+      const mockDeployImageData: DeployImageFormData = _.cloneDeep(mockDeployImageFormData);
+      const PORT1 = { containerPort: 8081, protocol: 'TCP' };
+      let serviceObj;
```

Comment:
do we know `type` of serviceObj? i think `K8sResourceKind` should work

Reply:
So in `K8sResourceKind` `podLabels` is of type `Selector` but from `getPodLabels` we are returning a simple object with the required label values  and not a `Selector` object. Hence if I assign `K8sResourceKind` it gives error. I tried to add a `Selector` Object with the required label values but while submitting the form it showed some error. That is why I have added any for now in the submit methods also for the resource object.